### ISSUE TITLE
Excessive location requests when not connected over Bluetooth...

### ISSUE THREAD

**Issue by kf6gpe** [ge0rg/aprsdroid#235]
I'm seeing this in the following scenario on Android Q beta. It may happen elsewhere.

1. Reboot phone.
1. Start APRSdroid, connect via Bluetooth to my TH-D74A. 
1. Use as normal for an hour.
1. Turn off the TH-D74A; Bluetooth silently disconnects. (I like that I can turn on the radio again and it automatically starts beaconing again.)
1. Wait a half day (my work day) and use the phone.
1. Battery life during this period is bad; a check of adb shell dumpsys location shows a lot of position activity by APRSdroid, even though it's not connected over Bluetooth.
1. Reconnect to Bluetooth by powering on my radio again, position continues updating as I expect.

I haven't looked at the code, but I'm thinking what happens is the Bluetooth stuff shuts down normally, but the location tracking does not in the case where the Bluetooth TNC is shut down.

**Comment by ge0rg** [comment]
Yeah, you are right. The location part and the RF part are completely separate, there is no feedback channel from the latter to the former. APRSdroid will also try to reconnect Bluetooth every 3 seconds, which is not battery friendly either.

I'm not sure how much work it would be to solve this issue, as I'd have to integrate a back-channel to all location sources and have some more logic to delay transmission after re-activation of GPS (it's typically very much off in the first 10~30 seconds).
